docs/commands/index.md

```diff
@@ -22,21 +24,39 @@ lenra_cli 0.0.0
 The Lenra command line interface
 
 USAGE:
-    lenra <SUBCOMMAND>
+    lenra [OPTIONS] <SUBCOMMAND>
 
 OPTIONS:
-    -h, --help       Print help information
-    -V, --version    Print version information
+        --config <CONFIG>    The app configuration file [default: lenra.yml]
+        --expose <EXPOSE>    Exposes services ports [possible values: app, devtool, postgres, mongo]
+    -h, --help               Print help information
+    -v, --verbose            Run the commands as verbose
+    -V, --version            Print version information
 
 SUBCOMMANDS:
-    build      Build your app in release mode
-    check      Checks the running app
-    dev        Start the app in an interactive mode
-    help       Print this message or the help of the given subcommand(s)
-    logs       View output from the containers
-    new        Create a new Lenra app project from a template
-    start      Start your app previously built with the build command
-    stop       Stop your app previously started with the start command
-    update     Update the tools Docker images
-    upgrade    Upgrade the app with the last template updates
+    build       Build your app in release mode
+    check       Checks the running app
+    dev         Start the app in an interactive mode
+    help        Print this message or the help of the given subcommand(s)
+    logs        View output from the containers
+    new         Create a new Lenra app project from a template
+    reload      Reload the app by rebuilding and restarting it
+    start       Start your app previously built with the build command
+    stop        Stop your app previously started with the start command
+    terminal    Start a Lenra command terminal to run commands with a same context (verbose,
```

Comment:
I'm not fan of the name `terminal` to launch this.

I think more natural to start the interactive mode when you just run `lenra` without any sub commands.
Currently it show the help, which is a subcommand `lenra help`.

What did you think about this @taorepoara ?

Reply:
I have changed to follow this way